From 08ff1a04abfde739f7b6d273ebfad65e9a09c24a Mon Sep 17 00:00:00 2001
From: James Le Cuirot <chewi@gentoo.org>
Date: Sun, 20 Mar 2022 13:19:49 +0000
Subject: [PATCH] Set odamast C++ standard to 98

Otherwise the use of `byte` is ambiguous with newer compilers, resulting
in a build error.

(cherry picked from commit 4c06d27e60b1e10c4d661523894430323398b140)
---
 master/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/master/CMakeLists.txt b/master/CMakeLists.txt
index cbe6a0311..f216eab9a 100644
--- a/master/CMakeLists.txt
+++ b/master/CMakeLists.txt
@@ -6,6 +6,7 @@ file(GLOB MASTER_SOURCES *.cpp *.h)
 # Master target
 add_executable(odamast ${MASTER_SOURCES})
 odamex_target_settings(odamast)
+set_property(TARGET odamast PROPERTY CXX_STANDARD 98)
 
 if(WIN32)
   target_link_libraries(odamast wsock32)
-- 
2.34.1

